### PR TITLE
Add meta og:description

### PR DESCRIPTION
**Pull request by wyzlle**

### CODE REVIEW

**Comment by yusufmalikul** [comment]
Any chance to review this? @tulir

**Comment by revi** [comment]
Personally it'd be much better to say something in line of "don't poke around to see someone, just ask your question in full right away" than just Java reference - I often use this page in non-programming environments and it'd be bit confusing in such cases.

**Comment by mtatko** [comment]
> Personally it'd be much better to say something in line of "don't poke around to see someone, just ask your question in full right away"

I have to agree with this one. The Java reference is there as an example of good and bad questions and might be confusing if you use the website in non-programming related discussion. The og:description should be something that makes a person immidiately think "oop, I should do this next time, instead of doing that".

**Review by mtatko — CHANGES_REQUESTED** [review]
> Personally it'd be much better to say something in line of "don't poke around to see someone, just ask your question in full right away"

I have to agree with this one. The Java reference is there as an example of good and bad questions and might be confusing if you use the website in non-programming related discussion. The og:description should be something that makes a person immidiately think "oop, I should do this next time, instead of doing that".

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta property="og:title" content="Don't ask to ask, just ask" />
   <meta property="og:type" content="website" />
   <meta property="og:url" content="https://dontasktoask.com" />
-  <meta property="og:description" content="Don't ask &quot;Any Java experts around?&quot;, but rather ask &quot;How do I do [problem] with Java and [other relevant info]?.&quot;" />
+  <meta property="og:description" content="Ask your questions directly instead of asking to ask. />
```
Comment (mtatko) [inline index.html:8]: What about something in line of `Hey, you! Don't poke around to see someone, ask your question in full right away. Thanks!`? In my opinion that seems way better than going full passive-agressive with "ask directly, don't ask to ask", however I'd love to hear what others think of that. I'm going to leave this as a comment for now.

Comment (mtatko) [reply index.html:8]: @revi what do you think?

**Review by mtatko — APPROVED** [review]

**Pull request closed without merging**